packages/client/src/ActionsAndReducers/dbWargames/wargames_ActionCreators.ts

```diff
@@ -430,3 +444,24 @@ export const updateWargameVisible = (dbName: string) => {
     dispatch(saveAllWargameNames(games))
   }
 }
+
+export const deleteAnnotation = (dbName: string, annotation: IconOption) => {
+  console.log('testDbName', dbName, 'testAn')

```

Comment:
@lilitkarapetyan - there are a few `console.log` entries that can be removed, once you're confident the code is working.